### ISSUE TITLE
Add Logging in the files 

### ISSUE THREAD

**Issue by joydeep2001** [CliqueLedger/Clique-Ledger-Core#62]
So far we are only using console.log to log. We don't have any persistent logs. We will use [winston](https://www.npmjs.com/package/winston) to add logs.  
How to Contribute to Logging

### Check for Existing Issues:

1. Before starting, search in the issue tab using the add-logging label to see if there’s already an open issue related to the file you want to work on.
2. If you find an existing issue for that file, please select another file to avoid duplication.

### Create a New Issue:

1. If no issue exists for the file, create a new one with the title: Add Logging in <file-name.ts> (replace <file-name.ts> with the actual file name).
2. Tag your issue with the add-logging label.

### Begin Implementation:
Once the issue is created and labeled, you can start working on adding logging using winston. Make sure to follow the guidelines for consistent logging.

**Note: Please follow these steps carefully to prevent multiple contributors from working on the same file, ensuring an organized and efficient process.**
